### PR TITLE
fix: test npm run build for docker services

### PR DESCRIPTION
**Pull request by Savio629**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: https://github.com/SamagraX-Stencil/stencil/issues/135

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/lib/service-temporal/service-temporal.factory.ts [hunk 1]
```diff
@@ -135,12 +135,12 @@ function addActivitiesImport(options: TemporalServiceOptions): Rule {
...
 
-    tree.create(envFilePath, content);
+//     tree.create(envFilePath, content);
 
-    return tree;
-  };
-}
+//     return tree;
+//   };
+// }
```
Comment (techsavvyash) [inline src/lib/service-temporal/service-temporal.factory.ts]: why remove this?

**Review by techsavvyash — CHANGES_REQUESTED** [review]

**Comment by Savio629** [comment]
@techsavvyash logging factory file used to have only code to generate the docker files.
So we don't require to call logging via the stencil generate command right?

**Comment by techsavvyash** [comment]
Yes @Savio629

**Comment by Savio629** [comment]
Opened a new pr #10

**Pull request closed without merging**